pennylane/ops/qubit/parametric_ops.py

```diff
@@ -170,7 +170,7 @@ def _matrix(cls, *params):
 
         p = qml.math.exp(-0.5j * theta)
 
-        return qml.math.diag([p, qml.math.conj(p)])
+        return qml.math.diag([p, 1/p])
```

Comment:
nice solution! With this change, does differentiating the matrix provide the expected result?

Reply:
yep!